dev.py

```diff
@@ -436,6 +436,13 @@ def setup_build(cls, dirs, args):
                 raise RuntimeError("Can't build into non-empty directory "
                                    f"'{build_dir.absolute()}'")
 
+        if sys.platform == "cygwin":
+            # Cygwin only has netlib lapack, but can link against
```

Comment:
We'll build this logic into `dependency('blas')` later, but for now this seems fine to add here.